std_msgs/Header header
string id
string model
string serial_num
AreaLimits overscan
AreaLimits effective
ChipInfo chip
uint32 bayer_format
bool is_color
bool is_cool
bool has_bin1x1_mode
bool has_bin2x2_mode
bool has_bin3x3_mode
bool has_bin4x4_mode
ParamLimits gain_limits
ParamLimits offset_limits
ParamLimits usb_traffic_limits
ParamLimits red_wb_limits
ParamLimits green_wb_limits
ParamLimits blue_wb_limits
ParamLimits temperature_limits
